### ISSUE TITLE
Update design: Clickable names

### ISSUE THREAD

**Issue by Brend-Smits** [DigitalExcellence/dex-frontend#279]
**Is your feature request related to a problem? Please describe.**
The designs need to be updated to reflect changes wanted in issue: https://github.com/DigitalExcellence/dex-frontend/issues/270 
This issue is specifically for the part about making the names clickable on the project detail page and on the project overview page. 

**Describe the solution you'd like**
Updated Adobe XD files

**Comment by juliaslaats** [comment]
![image](https://user-images.githubusercontent.com/71012148/97288437-20ae4000-1846-11eb-95a9-224dd3960981.png)
![image](https://user-images.githubusercontent.com/71012148/97288465-286de480-1846-11eb-8123-a6f8e14b2e97.png)

Inspired by a couple other places, I decided to make the background change color when you hover above the name, to illustrate that the name is clickable. I also changed the font to a darker color than the date and the description, to show that it is not just plain text. Any feedback is welcome!
